extensions/ql-vscode/src/model-editor/model-evaluation-run.ts

```diff
@@ -1,4 +1,5 @@
 export interface ModelEvaluationRun {
   isPreparing: boolean;
   variantAnalysisId: number | undefined;
+  isModelAlertsViewOpen?: boolean;
```

Comment:
Shall we make this a non-optional boolean to avoid having to deal with `undefined`?

I'm in two minds about whether the `ModelEvaluationRun` domain entity is the right place to hold information around whether a view is open or not. It feels a bit wrong because it's not a property of the evaluation run, just some information around the user's modeling setup. So it could be information we have in the `InternalDbModelingState` instead. Any thoughts?

Reply:
I think I agree that `ModelEvaluationRun` feels wrong 🤔 I hadn't thought of `InternalDbModelingState`, but will try adding it there instead!